src/components/walkthroughResources/walkthroughResources.js

```diff
@@ -63,10 +63,17 @@ class WalkthroughResources extends React.Component {
     return provisioningStatus;
   }
 
+  renderHeading() {
+    if (!this.props.noHeadline) {
+      return <h4 className="integr8ly-helpful-links-heading">Walkthrough Resources</h4>;
```

Comment:
Should this still be displayed? In my mind it looks a bit strange to have this heading and then the other headings? @finp @david-martin Any opinions?